lib/game_board.rb

```diff
@@ -0,0 +1,67 @@
+class GameBoard
+  attr_accessor :player1, :player2
```

Comment:
Please use attr_reader for these attributes. They are not set anywhere after the GameBoard is initialized.
Same goes for the attributes in your Player class.